### PR TITLE
Uxd 860 input remove type number

### PR DESCRIPTION
**Pull request by tristanjasper**
### Purpose 🚀
Remove `Number` Input type
Remove unneeded custom prop

### Notes ✏️
_details of code change / secondary purposes of this PR_

### Updates 📦
- [x] MAJOR (breaking) change to CustomPropTypes, Input (if the package you are breaking is a dependency in other packages, be sure to add a commit with a "BREAKING CHANGE" message to those other packages as well)
- [ ] MINOR (backward compatible) change to _these packages_
- [ ] PATCH (bug fix) change to _these packages_

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/your-branch-name

### Screenshots 📸
_optional but highly recommended_

### References 🔗
_relevant Jira ticket / GitHub issues_

### CODE REVIEW

File: packages/Input/stories/examples/Types.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ const TypesExampleStory = () => {
           <code>type=&quot;number&quot;</code>
         </small>
       </h3>
-      Don&apos;t do this. Use the &lt;InputNumber&gt; component.
```
Comment (tristanjasper) [inline packages/Input/stories/examples/Types.js:27]: Not sure what this <InputNumber> component is referring to

Comment (allison-c) [reply packages/Input/stories/examples/Types.js:27]: I believe it's referring to the ACL UI InputNumber haha

**Review by allison-c — APPROVED** [review]
LGTM for removing the number type, any plan to support it again in paprika through `text` type?

**Comment by mikrotron** [comment]
> LGTM for removing the number type, any plan to support it again in paprika through `text` type?

Maybe we can migrate this from Robots:
https://github.com/acl-services/skynet-server/blob/integration/app/javascript/robots/app/pages/Tasks/TaskDesigner/Analytics/components/Widgets/NumericInput/NumericInput.js

**Comment by tristanjasper** [comment]
> > LGTM for removing the number type, any plan to support it again in paprika through `text` type?
> 
> Maybe we can migrate this from Robots:
> https://github.com/acl-services/skynet-server/blob/integration/app/javascript/robots/app/pages/Tasks/TaskDesigner/Analytics/components/Widgets/NumericInput/NumericInput.js

I'd be happy to do that, perhaps in a subsequent pr. I wasn't aware we had that component, but looks good

**Comment by tristanjasper** [comment]
created issue https://github.com/acl-services/paprika/issues/955

**Comment by mikrotron** [comment]
> I'd be happy to do that, perhaps in a subsequent pr. I wasn't aware we had that component, but looks good

Thanks for creating the issue, @tristanjasper! 👍

**Pull request merged**